README.md

```diff
@@ -1349,6 +1349,7 @@ In addition to manually setting environment variables, cargo-make will also auto
 * **CARGO_MAKE_RUST_TARGET_OS** - windows, macos, ios, linux, android, etc ... (see rust cfg feature)
 * **CARGO_MAKE_RUST_TARGET_POINTER_WIDTH** - 32, 64
 * **CARGO_MAKE_RUST_TARGET_VENDOR** - apple, pc, unknown
+* **CARGO_MAKE_RUST_TARGET_TRIPLE** - x86_64-unknown-linux-gnu, x86_64-apple-darwin, x86_64-pc-windows-msvc, etc ...
```

Comment:
the readme is actually generated from content.md (and few others) by running cargo make generate-readme
so doing a change to this file only won't help.
better to add this line to content.md (in docs/_includes) and run cargo make generate-readme

Reply:
I added it, sadly I can't run `generate-readme` on my machine because I am currently running windows 😞.